CHANGELOG.md

```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+# [5.13.4] - 2024-06-11
+
+### Added
```

Comment:
Because you are adding something, can you update the version to 5.14.0 instead?

Reply:
Good call, version updated to 5.14.0.